package.json

```diff
@@ -97,7 +97,7 @@
     "prettier-eslint": "15.0.1",
     "prettier-eslint-cli": "7.1.0",
     "replace-in-file": "^7.0.1",
-    "semantic-release": "^24.2.0",
```

Comment:
I am OK with falling back for now, but being stuck on 22 isn't good either. Did they list something in the breaking changes around this?